src/main/java/frc/robot/RobotContainer.java

```diff
@@ -4,13 +4,22 @@
 
 package frc.robot;
 
+import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj2.command.Command;
-import frc.robot.commands.autoCmd;
-import frc.robot.commands.teleopCmd;
+import frc.robot.commands.basic.autoCmd;
+import frc.robot.commands.basic.teleopCmd;
 import frc.robot.subsystems.drivetrainSubsystem;
 
 public class RobotContainer {
+  // Defining Controllers
+  private final Joystick driver1 = new Joystick(Constants.DRIVER_ONE_PORT);
+  private final Joystick driver2 = new Joystick(Constants.DRIVER_SEC_PORT);
+  private final Joystick operator = new Joystick(Constants.OPER_PORT);
```

Comment:
These don't look like they're being used. (compared to the ones in teleopCmd.java)

Reply:
They are set aside for when the actual code is constructed, just a placeholder